windows-iotcore/develop-your-app/MultiLang.md

```diff
@@ -26,12 +26,96 @@ See the below sample applications:
 
 ## Languages in OS
 
-Windows 10 IoTCore kits now include the language resources for the following languages
-
-* English : en-US, en-GB
-* French  : fr-FR, fr-CA
-* Spanish : es-ES, es-MX
-* Chinese : zh-CN
+Windows 10 IoTCore kits now include the language resources for the following languages:
+
+<table>
+<colgroup>
+<col width="50%" />
+<col width="50%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th align="left">Language</th>
+<th align="left">Code</th>
+<th align="left">Region</th>
+</tr>
+</thead>
+<tbody>
+
```

Comment:
Shouldn't this be using MD syntax instead of HTML?

Reply:
You are absolutely right! Changing this now...